avocado_vt/test.py

```diff
@@ -263,22 +239,15 @@ def runTest(self):
         reports the correct results
         """
         if self.__status != "PASS":
-            if isinstance(self.__status, error.TestNAError):
-                self.cancel(str(self.__status))
-            elif isinstance(self.__status, error.TestWarn):
-                self.log.warn(str(self.__status))
-            elif isinstance(self.__status, error.TestFail):
-                self.fail(str(self.__status))
-            else:
-                raise self.__status  # pylint: disable=E0702
```

Comment:
why remove the judgement here?

Reply:
runtest()actually not called in this py file,but _runtest() was used instead, not quite sure whether it is useful or not. So better change them to exceptions.TestError,exceptions.TestFail, exceptions.TestWarn respectively

Reply:
As autotest has been obsoleted, so this section is deleted. But some tp-libvirt tests still need autotest, freeze this pr until tp-libvirt update or freeze the avocado-vt tag